docs/sources/_index.md

```diff
@@ -105,6 +105,21 @@ Grafana Agent can ingest OpenTelemetry, OpenCensus, Jaeger, Zipkin, or Kafka spa
 For more information on how to configure, refer to [receivers]({{< relref "./configuration/traces-config.md" >}}).
 The Grafana Agent is also capable of exporting to any OpenTelemetry GRPC compatible system.
 
+## Supported platforms
+
+* Linux
+  * Minimum version: kernel 2.6.32 or later
+  * Architectures: AMD64, ARM64, ARMv6, ARMv7
+* Windows
+  * Minimum version: Windows Server 2008R2 or later, or Windows 7 or later.
```

Comment:
IMO this should be `Windows Server 2012` which is supported through 2023 and `Windows 10` or later. Windows 8.1 support ends in January. It will work on older versions but I think there is a difference in supported and it works.